Code/Java/Tower_of_Hanoi.java

```diff
@@ -0,0 +1,83 @@
+
+/**
+ * Tower of Hanoi. Recursive method is used.
+ * Author - Khushee Kapoor
```

Comment:
Being in open source, we make contributions to help wider community. So, plz remove this line :)

Reply:
Changes made